Feature/ot238/51 52 53

pableArg opened the pull request.

pableArg committed:
Init feature

pableArg committed:
Init feature

pableArg committed:
Init features

pableArg committed:
In

pableArg committed:
No terminado

Franpepper committed:
Added @SerializedName to ActivitiesResponse
Added Height_percent attribute to ImageView on fragment_activities_item

pableArg committed:
End ticket 51-52-53

pableArg committed:
Merge branch 'main' into feature/OT238/51-52-53

# Conflicts:
#	app/src/main/res/layout/us_member_detail.xml

maxwellnewage commented on app/src/main/java/com/melvin/ongandroid/model/entities/activities/Activitie.kt:
```diff
@@ -0,0 +1,14 @@
+
+import android.text.Html
+
+data class Activitie(
```
Singular: Activity
Plural: Activities

maxwellnewage commented on app/src/main/java/com/melvin/ongandroid/model/entities/activities/Activitie.kt:
```diff
@@ -0,0 +1,14 @@
+    val image: String,
+){
+    val rawDescription: String
+        get() = Html.fromHtml(this.description, Html.FROM_HTML_MODE_COMPACT).toString()
```
Mirar esto: https://stackoverflow.com/questions/37904739/html-fromhtml-deprecated-in-android-n

maxwellnewage commented on app/src/main/java/com/melvin/ongandroid/model/entities/activities/ActivitiesResponse.kt:
```diff
@@ -0,0 +1,10 @@
+data class ActivitiesResponse (
+    val success: Boolean,
+    @SerializedName("data")
+    val activitieList: List<Activitie>
```
activitiesList

maxwellnewage commented on app/src/main/java/com/melvin/ongandroid/view/fragment/LoginFragment.kt:
```diff
@@ -23,7 +22,7 @@ import dagger.hilt.android.AndroidEntryPoint
     private lateinit var _binding : FragmentLogInBinding
     private val loginBinding get() = _binding
-    private val loginViewModel: LogInViewModel by viewModels()
+    private val loginViewModel : LogInViewModel by viewModels()
```
revisar esto

maxwellnewage commented on app/src/main/res/layout/fragment_activitie.xml:
```diff
@@ -0,0 +1,32 @@
+
+
+    <ProgressBar
+        android:id="@+id/progressSearch"
```
no seria pbSearch?

maxwellnewage commented on app/src/main/res/values/strings.xml:
```diff
@@ -43,7 +43,8 @@
     <string name="textError"> Error al cargar los slides </string>
-    <string name="actionText" > RETRY</string>
+    <string name="txtErrorActivites"> Ha ocurrido un error obteniendo la info </string>
+    <string name="actionText"> RETRY</string>
```
por que hay un espacio en RETRY?

pableArg committed:
- Se renombro todas las definicione de activity por WhatWeD.
- Se arreglaron el espacio en los strings.xml
- se renombro el id del progress bar

pableArg committed:
Merge branch 'main' into feature/OT238/51-52-53
